packages/calcite-components/src/components/chip/chip.tsx

```diff
@@ -413,7 +413,6 @@ export class Chip
                 ? toAriaBoolean(this.selected)
                 : undefined
             }
-            aria-disabled={disableInteraction ? toAriaBoolean(disabled) : undefined}
```

Comment:
This one was only setting `aria-disabled` when `disabled` was true anyway. Not sure why the `disabledInteraction` is being tested here unless we do this:

```tsx
 aria-disabled={disableInteraction ? toAriaBoolean(disableInteraction) : undefined}
```